wikimedia/wikipedia-ios
# kick thankyou.wikipedia.org links out to web browser

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T259312

### Notes
* To create a beta version of the app off of, to share w/ advancement team.

### Test Steps
1. Run app.
1. Close app if you want.
1. Open this URL by dropping it in a note and then tapping the link: https://thankyou.wikipedia.org/wiki/Thank_You/en?country=US" (If in simulator, run `xcrun simctl openurl booted "https://thankyou.wikipedia.org/wiki/Thank_You/en?country=US"`.)
1. Ensure it opens in the default web browser. (It will open the app for a moment.)
